### DISCUSSION TITLE
Quasar Developer Meetups

### DISCUSSION THREAD

**Discussion by hawkeye64** [quasarframework/quasar#9658] [Ideas / Proposals]
Not a bug/issue/feature request, but a discussion area.

If you would like a Quasar meetup please list your top 3 areas?

For instance, mine would be:
1. Calgary (Canada)
2. Vancouver (Canada)
3. Toronto (Canada)

I am not saying this will happen in your location, but maybe we can get some community driven ones based on the highest concentration.

**Comment by ibrainventures** [comment]
YEAH!! 
Some month ago i had a short discussion inside discord and there where some interes signaled by some other quasarians. 

The overlap "webdeveloper + vue + quasar + time / interest meetup" with a regional aspect is maybe to small for geeting here (southwest germany - frankfurt / stuttgart / munich) some quasarians..

So the idea was to go more the direction "european centered meetup" (here was one idea prague).

If there are some southwest-germany quasar boys/girls interrested for a meetup, please give a sign.

**Update** (added the ordered preference list)

Frankfurt, Bucharest, Palma, Prague, Berlin

**Comment by hawkeye64** [comment]
@ibrainventures And, your top 3 locations?

**Comment by smolinari** [comment]
In order of preference....Frankfurt, Berlin, Bucharest, 

Scott

**Comment by valasek** [comment]
In order of preference .. Prague, Berlin, Frankfurt, Bucharest
Stanislav

**Comment by IlCallo** [comment]
In my company we are thinking about start organizing local meetups in next years, and eventually end up with a conf.
Not so easy, because we are in Reggio Emilia (Italy) and there are few programmers like us, but we would like to gather people from nearby cities.

We'd definitely participate to an european meetup/conf, for us Milan (Italy), Berlin (Germany), Prague (Czechia) and Amsterdam (Netherlands) are all viable locations (Milan or Prague would be the less expensive I think)

**Comment by ibrainventures** [comment]
@hawkeye64 
i updated my entry. May this "issue" can be pinned?

**Comment by m0jimo** [comment]
Prague
Berlin
Vienna

**Comment by lucasfernog** [comment]
São Paulo

**Comment by binarytide** [comment]
shoutout for Boston!

**Comment by richardkeep** [comment]
Nairobi 😁

**Comment by tohagan** [comment]
Brisbane, Australia

**Comment by codenamezjames** [comment]
Orlando, FL
Atlanta, GA
Tampa, Fl

**Comment by JeepayJipex** [comment]
Paris, 
Amsterdam, 
London
😁

**Comment by valasek** [comment]
Here we are - https://www.google.com/maps/d/viewer?mid=1JnR2TKx6ej_4RMmMHYGNoa4ePokhv-u9&hl=en&usp=sharing

![image](https://user-images.githubusercontent.com/1046787/71765622-81573f80-2ef7-11ea-990b-508dfad858a9.png)

**Comment by patrickmonteiro** [comment]
São Paulo

**Comment by smakinson** [comment]
Orlando

**Comment by valasek** [comment]
Map updated - https://www.google.com/maps/d/viewer?mid=1JnR2TKx6ej_4RMmMHYGNoa4ePokhv-u9&hl=en&usp=sharing

**Comment by bloodf** [comment]
São Paulo

**Comment by jagadishallakanti** [comment]
Melbourne

**Comment by gregveres** [comment]
Waterloo, Ontario, Canada

**Comment by pratik227** [comment]
Pune (India)

**Comment by valasek** [comment]
Map updated - https://www.google.com/maps/d/viewer?mid=1JnR2TKx6ej_4RMmMHYGNoa4ePokhv-u9&hl=en&usp=sharing

I know a JS developer/speaker who might help me to organize a meetup in Prague - https://www.codinglawyer.io/.

In Prague we can:
- organize own meetup, or join
- PragueJS - https://www.meetup.com/praguejs/ (organized once a month, no plan for January yet) 
- or Vue.js CZ/SK - https://www.meetup.com/vuejs-cz-sk/, next meeting on Jan / 22

**Comment by bloodf** [comment]
In São Paulo we have:

Vue.JS SP (Meetup): https://www.meetup.com/pt-BR/vuejssp/
JS SP (Meetup): https://www.meetup.com/pt-BR/Javascript-SP/

We can create a specific for Quasar if there are many interested persons.

I'm currently changing job so that I will implement Quasar on a new company, and we will have more Quasarians being born in São Paulo :)

**Comment by nikoz84** [comment]
> São Paulo
> Salvador Bahia

**Comment by digitalextremist** [comment]
I'd participate in something in `Southern California` such as `San Diego` or `Los Angeles` especially.

**Comment by fergusmeiklejohn** [comment]
I'm in Taiwan so I'd like a virtual meetup and/or everything to be filmed

**Comment by jesusvilla** [comment]
Lima (Perú)

**Comment by valasek** [comment]
Thank you all for yor votes.

Map was updated: https://www.google.com/maps/d/viewer?mid=1JnR2TKx6ej_4RMmMHYGNoa4ePokhv-u9&hl=en&usp=sharing

![image](https://user-images.githubusercontent.com/1046787/72389586-0678fa80-3729-11ea-955a-5b4ae17774ab.png)

**Comment by maddeezy** [comment]
Southern California

Los Angeles
Orange County
San Diego

**Comment by KevinYang0120** [comment]
Taiwan

**Comment by jvaca92** [comment]
Prague

**Comment by eladcandroid** [comment]
Here is my Quasar lecture on a meetup as part of Vue.js Israel Community:

https://www.meetup.com/vue-js/events/266895998/

![vue](https://user-images.githubusercontent.com/13297000/72596801-24a34e00-3915-11ea-824e-81d15fc6efb7.jpg)

**Comment by jiyeonseo** [comment]
Seoul (South Korea 🇰🇷)

**Comment by dariodepaolis** [comment]
Naples (we have the best pizza!) <3

**Comment by stevebor1** [comment]
Orange County California!
- Irvine
- Anaheim
- Costa Mesa

**Comment by maddeezy** [comment]
@stevebor1 Let's set one up for the OC

**Comment by maddeezy** [comment]
@digitalextremist interested in the OC as well?

**Comment by digitalextremist** [comment]
@maddeezy and @stevebor1 I'd come up to the `OC`, which does seem fair vs. `LA` or `SD` :smile:

**Comment by valasek** [comment]
I'd like to encourage you to organize local meetups. Let us know when you plan to do so ideally a week or two ahead. We will reach out to the Quasar community and ask Quasar core members to participate as well. Just tag me on DIscord or DM me.

Map has been updated: https://www.google.com/maps/d/viewer?mid=1JnR2TKx6ej_4RMmMHYGNoa4ePokhv-u9&hl=en&usp=sharing

![Screenshot from 2020-01-28 07-49-58](https://user-images.githubusercontent.com/1046787/73241676-0229f400-41a3-11ea-84bf-0e1008298103.png)

**Comment by maddeezy** [comment]
@stevebor1 @digitalextremist 

I'll actually be in SD next week.

I see a few proposed meetup locations on that Map...I'm down for whenever/wherever LA->SD

Looking forward to it

**Comment by valasek** [comment]
@maddeezy, @stefanvanherwijnen, @digitalextremist we have created a [Discord conferences channel](https://discordapp.com/channels/415874313728688138/671632517962661888) where I encourage you to post the link to the planned event so we can share with the Quasar community.

**Comment by dantrevino** [comment]
Portland, OR

**Comment by maucrvlh** [comment]
São Paulo, BR

**Comment by tanthammar** [comment]
Amsterdam

**Comment by diadal** [comment]
Lagos Nigeria

**Comment by aleksander237** [comment]
Cracow, Prague, Bratislava

**Comment by martijn788** [comment]
Amsterdam
London
Paris

**Comment by marekkaczkowski** [comment]
Wroclaw, Warsaw, Cracow

**Comment by tech-andgar** [comment]
Bogotá, Colombia

**Comment by MonchiLin** [comment]
Shenzhen（China）
Shanghai（China）

**Comment by tohagan** [comment]
Since we're almost all in lock-down, how 'bout we do a zoom virtual meetup instead?

**Comment by maddeezy** [comment]
Sure...although seen a lot of Zoom vulnerabilities lately...


-------- Original Message --------

**Comment by jdus** [comment]
Vienna

Would travel though, i.e. to Prague, when all this is over.

**Comment by tohagan** [comment]
Zoom has acted on the feedback and done a stash of stuff. 
- https://blog.zoom.us/wordpress/2020/04/01/a-message-to-our-users/

There are also security guidelines to [prevent Zoom bombing](https://www.fbi.gov/contact-us/field-offices/boston/news/press-releases/fbi-warns-of-teleconferencing-and-online-classroom-hijacking-during-covid-19-pandemic).  One of them is "don't share a link publicly" as in ... not [in a GitHub issue](https://github.com/quasarframework/quasar/issues/5983#issuecomment-610731000) 😄 .

**Comment by maddeezy** [comment]
@tohagan @hawkeye64 

Do we have a discord channel set up for meetups?

**Comment by hawkeye64** [comment]
@maddeezy You can use the #conferences channel for meetups as well

**Comment by MichaelNino** [comment]
Orlando, FL
Atlanta, GA
Tampa, Fl

**Comment by Mansehej** [comment]
New Delhi, India
Chandigarh, India
Bengaluru, India

**Comment by DmitrijOkeanij** [comment]
May be somewhere not only to meetup, but also to rest.
Meetup on nature ground?
May be location with lake, see, forest.
I think place to meetup quality is not less important than map location.

**Comment by hawkeye64** [comment]
Interesting you should say that @DmitrijOkeanij as I have had conversation with a Quasar developer who owns property and has invited us to host there. Check it out: https://www.wasalakeresort.com/

**Comment by smolinari** [comment]
We're going to have an online conference. 

See the details here: https://dev.to/quasar/introducing-quasar-conf-52gg

Scott

**Comment by facku** [comment]
Is there going to be a live streaming for those of us who live at the end of the world? I am from Argentina, but even if it will done in BsAs, it is still far away.

**Comment by hawkeye64** [comment]
@facku I live in Canada and it will be 09:00am my time. Unless you like sleeping in, you should not have a bad time (like 01:00am)

**Comment by webnoob** [comment]
Video now available for our first live conf: https://www.youtube.com/watch?v=6ZKBZ3k4Ebk

**Comment by MAMUGI1492** [comment]
Madrid

**Comment by dirkhe1051931999** [comment]
xi'an China

**Comment by SaulJWu** [comment]
> xi'an China

HaHa
